dbmongo/js/public/map.ts

```diff
@@ -80,9 +81,20 @@ export function map(this: Input): void {
     const v: Partial<SortieMapEntreprise> = {}
     const diane = f.diane(value.diane)
     const bdf = f.bdf(value.bdf)
-    const sirene_ul = (value.sirene_ul || {})[
-      Object.keys(value.sirene_ul || {})[0] || ""
-    ]
+    const sirene_ul =
+      f.iterable(value.sirene_ul)[0] ||
+      ((null as unknown) as EntréeSireneEntreprise)
```

Comment:
Je ne comprends pas pourquoi tu fais une double conversion de types ici. J'ai l'impression que cette écriture suffit:

```suggestion
    const sirene_ul = f.iterable(value.sirene_ul)[0] || null
```

Reply:
Ahah effectivement, j'avais pas écrit ça comme ça au début, et le linter criait un peu d'où la double conversion qu'il m'imposait pour instancier directement sirene_ul à null mais au type EntréeSireneEntreprise, mais maintenant ce que tu proposes marche parfaitement bien.
Merci !